custom-recipes/timeseries-preparation-extrema-extraction/recipe.py

```diff
@@ -1,22 +1,28 @@
 # -*- coding: utf-8 -*-
 import logging
+import sys
+
 from dataiku.customrecipe import *
-from dku_timeseries import ExtremaExtractor
-from commons import *
 
+from commons import *
+from dku_timeseries import ExtremaExtractor
 
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO, format='timeseries-preparation plugin %(levelname)s - %(message)s')
 
+if sys.version_info.major == 2:
+    logger.warning("You are using Python {}.{}. Python 2 is now deprecated for the Time Series preparation plugin. Please consider creating a new Python 3.6 "
+                   "code environment".format(sys.version_info.major, sys.version_info.minor))
+
```

Comment:
can you put all the code env warnings of the recipe in a unique function

Reply:
done in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/15/commits/20854064b29a1858a9c273425ed449acee21b41a